reference/dic_tags.rst

```diff
@@ -362,6 +363,18 @@ console.command
 For details on registering your own commands in the service container, read
 :doc:`/console/commands_as_services`.
 
+container.hot_path
+------------------
+
+**Purpose**: Add to list off always needed services
+
+This tag identifies the services that are always needed. It is only applied to a very short list of bootstrapping services (like ``router``, ``event_dispatcher``, ``http_kernel``, ``request_stack``, etc...). 
```

Comment:
Instead of having long lines wrapped by the text editors, we use a 80-character limit per lines (not a hard limit, we can surpass it a bit if needed). But don't worry, mergers can do that while merging your contribution.